### ISSUE TITLE
fresh 0.4.6 Server will not start

### ISSUE THREAD

**Issue by natebc** [Chuunkyz/BenchmarkII-1.18#22]
Hit the following crash trying to update my existing server but did a clean repro to generate this log. Happy to help debug if there's any more info you need.

```
[13:11:53] [main/FATAL]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'spectrum'!
        at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.12) ~[?:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.12.12.jar:?]
        at net.minecraft.server.Main.main(Main:106) [server-intermediary.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.12.12.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_437
        at de.dafuqs.spectrum.registries.SpectrumItemGroups$1.<init>(SpectrumItemGroups.java:21) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups.<clinit>(SpectrumItemGroups.java:20) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumBlocks.<clinit>(SpectrumBlocks.java:87) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.SpectrumCommon.onInitialize(SpectrumCommon.java:88) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 15 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_437
        at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
        at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:158) ~[fabric-loader-0.12.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups$1.<init>(SpectrumItemGroups.java:21) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumItemGroups.<clinit>(SpectrumItemGroups.java:20) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.registries.SpectrumBlocks.<clinit>(SpectrumBlocks.java:87) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at de.dafuqs.spectrum.SpectrumCommon.onInitialize(SpectrumCommon.java:88) ~[spectrum-1.1.5-1.18.1-earlygame6.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 15 more
```
gist of server-setup-config.yaml and latest.log: https://gist.github.com/natebc/002206f114b370bf0b332bc46b95c1dd

**Comment by natebc** [comment]
fwiw i was able to get this to load up by removing ` mods/spectrum-1.1.5-1.18.1-earlygame6.jar` based on some googling I did of the Caused by in the stack trace. I haven't looked up that mod but I suppose maybe it's a client-only mod?

**Comment by natebc** [comment]
Seems that's not a client-side only mod but may have some client-side pieces that are getting loaded in the server. Reported to the mod owner's github as well.

**Comment by natebc** [comment]
This does seem to be resolved in version 0.4.7, thanks!
